frontend/src/util/LiveMessageManager.js

```diff
@@ -70,7 +73,11 @@ export class LiveMessageManager {
     this.closeEventSourceConnection()
 
     this.eventSource = new EventSource(
-      `${url}/users/${this.userId}/live_messages`,
+      `${url}/users/${this.userId}/live_messages` + (
+        this.lastEventId
+          ? `?last_event_id=${this.lastEventId}`
+          : ''
+      ),
```

Comment:
I think those changes are not part of this issue…

Reply:
right, I reverted this.